docs/QuadraticSolver.puml

```diff
@@ -0,0 +1,33 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+activate Command
+Command -> Quadratic: quadratic.solveQuadratic()
+
+activate Quadratic
+Quadratic -> Quadratic: findA()
+activate Quadratic #FFBBBB
+deactivate Quadratic
+Quadratic -> Quadratic: findB()
+activate Quadratic #FFBBBB
+deactivate Quadratic
+Quadratic -> Quadratic: findC()
+activate Quadratic #FFBBBB
+deactivate Quadratic
+Quadratic -> Quadratic: quadraticFormula()
+activate Quadratic #FFBBBB
```

Comment:
Perhaps you can adjust the method call arrows for `findA()`, `findB()`, `findC()` and `quadraticFormula()` such that they point to the start of the corresponding activation bars, like what they arrow for `printAnswer()` does